config/env/internal-external.json

```diff
@@ -1,20 +1,21 @@
 {
   "services": {
     "ceramic": {
+      "launchLocation": "jest",
```

Comment:
This `launchLocation` concept seems redundant of `mode`.  Can we consolidate them?

EDIT: See my comment below

Reply:
Originally I was going to make this a boolean so it's probably better to do something like `preLaunched: true` and then use mode for whatever is inprocess. But yeah mode does not serve same purpose.